modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonCodegen.java

```diff
@@ -655,14 +655,14 @@ public String toModelName(String name) {
 
         // model name cannot use reserved keyword, e.g. return
         if (isReservedWord(camelizedName)) {
-            String modelName = "model_" + camelizedName; // e.g. return => ModelReturn (after camelize)
+            String modelName = "Model_" + camelizedName; // e.g. return => ModelReturn (after camelize)
```

Comment:
What do you think about removing the underscore? It's not a normal character for class names in python.

Reply:
You're right.